### ISSUE TITLE
Integration with TF ObjectDetection

### ISSUE THREAD

**Issue by AlexDag** [Labelbox/Labelbox#43]

**Comment by Riegerb** [comment]
Hey @AlexDag , are you referring to integrating with this: https://github.com/tensorflow/models/tree/master/research/object_detection  ?

**Comment by AlexDag** [comment]
I've already rewrite my local labelbox2pascal.py to get "TF object detection" style xml annotation files.
The one important thing: one annot. xml file must has only one type on Label, one or more but only one Label. There are some more issues,  as  for instance, inside xml image file name must be as Label name too.

**Comment by AlexDag** [comment]
hi , yes, I meant this framework.

**Comment by Riegerb** [comment]
OK - email to: brian@labelbox.io

**Comment by manuaero** [comment]
@AlexDag 
We've made it simpler for you to export your training data from Labelbox into Tensorflow. Now you can export your labeled images in Tensorflow's binary format (.tfrecord). This feature is currently in beta and we'd love to hear how we can further improve this to save you more time.
